cmake/CMakeLists.txt

```diff
@@ -1483,6 +1474,26 @@ install(
   DESTINATION ${CMAKE_INSTALL_SYSCONFDIR}/profile.d
 )
 
+###############################################################################
+# Install LAMMPS module into site-packages folder
+# Only available, if a shared library is built
+###############################################################################
+if(BUILD_LIB AND BUILD_SHARED_LIBS)
```

Comment:
Should that be behind `if(PKG_PYTHON)`? Also I was always wondering if `PKG_PYTHON` even made sense with `BUILD_LIB=OFF` or `BUILD_SHARED_LIBS=OFF`.

Reply:
`PKG_PYTHON` and the LAMMPS python module are two different things. The python module provides a LAMMPS module to python and uses ctypes to interface functions from the library interface imported from a shared library. The PYTHON package embeds the python interpreter into LAMMPS. Now it **is** possible to use both at the same time, but that is rather the exception than the rule.
Yes, `PKG_PYTHON` makes a lot of sense without building libraries. I do it all the time.

TL;DR: `PKG_PYTHON` allows to run python code inside of LAMMPS, `lammps.py` allows to run LAMMPS inside of python.